include/openssl/param_build.h

```diff
@@ -39,38 +39,38 @@ typedef struct {
     OSSL_PARAM_BLD_DEF params[OSSL_PARAM_BLD_MAX];
 } OSSL_PARAM_BLD;
```

Comment:
Are you sure the structures are stable enough for public view for the foreseeable future?

Reply:
See the comment:

> This currently exposes the param builder struct. It's probably worth the effort making this opaque.

So no, I don't think they should be public.  They are probably stable enough to be since OSSL_PARAM isn't going to change.

Reply:
The question at this point is should the param builder be public or not?

We're using it quite a bit in the providers.